Packs/Indeni/Integrations/Indeni/Indeni.yml

```diff
@@ -0,0 +1,207 @@
+category: Network Security
+commonfields:
+  id: Indeni
+  version: -1
+configuration:
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- additionalinfo: If true, only Palo Alto Networks Vulnerability issues will be pulled.
+    If false, all issues will be pulled.
+  defaultvalue: 'true'
+  display: Only Pull Palo Alto Networks Vulnerability Issues
+  name: onlyPullPanCveIssues
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: CRITICAL
+  display: Lowest Issue Severity To Pull
+  hidden: false
+  name: issueSeverity
+  options:
+  - CRITICAL
+  - ' ERROR'
+  - ' WARN'
+  - ' INFO'
```

Comment:
looks like you added space to the strings. I think this should be:
```
options:
- CRITICAL
- ERROR
- WARN
- INFO
```

Reply:
Done